app/frontend/app/controllers/pro.js

```diff
@@ -104,7 +103,9 @@ export default Ember.ArrayController.extend(HasCurrentUser, {
         key: PreloadStore.get('stripe_key'),
         name: "Hummingbird PRO",
         description: this.get('selectedPlan.name'),
-        token: tokenCallback
+        token: tokenCallback,
+        amount: parseInt(this.get('selectedPlan.amount') * 100),
```

Comment:
Then `parseFloat`?

The parseInt is probably receiving a Number from the multiplication already and doing nothing except ensuring it's an integer (or converting Number -> String -> Number, I'm not sure how parseInt works internally).  If it's supposed to truncate, then `Math.floor` is probably more idiomatic?